ci/collect_logs.yml

```diff
@@ -0,0 +1,38 @@
+---
+- hosts: all
+  name: "Create zuul-output log dir"
+  gather_facts: false
+  tasks:
+    - name: "Create log dir"
+      ansible.builtin.file:
+        path: "{{ ansible_user_dir }}/zuul-output/logs"
+        state: directory
+        mode: "0755"
+
+- hosts: localhost
+  name: Collect logs on the controller
+  gather_facts: false
+  tasks:
+    - name: Create log dir
```

Comment:
I don't know how much Zuul likes things being created on the executer node.

It seems that this does work, so perhaps we can update the output_dir for the custom_logger to put the files directly into the {{ ansible_user_dir }}/zuul-output/logs/controller/ dir.

This needs some investigation, i.e. is the output dir hardcoded in the plugin, and how can it be updated?